ingestion-beam/src/main/java/com/mozilla/telemetry/io/Read.java

```diff
@@ -128,8 +128,13 @@ public BigQueryInput(String tableSpec, BigQueryReadMethod method, Source source,
             // We have to take care not to read additional bytes; see
             // https://github.com/mozilla/gcp-ingestion/issues/1266
             ByteBuffer byteBuffer = (ByteBuffer) record.get(FieldName.PAYLOAD);
-            byte[] payload = new byte[byteBuffer.limit()];
-            byteBuffer.get(payload);
+            final byte[] payload;
+            if (byteBuffer != null) {
+              payload = new byte[byteBuffer.limit()];
+              byteBuffer.get(payload);
+            } else {
+              payload = new byte[0];
+            }
```

Comment:
Added this code to handle null payload for backfill taking place in https://github.com/mozilla/bigquery-backfill/pull/15